index.html

```diff
@@ -173,7 +173,8 @@ <h2 class="contact-title">Contact me</h2>
             action="https://formspree.io/f/xpznvyer"
             method="POST"
             class="contact-form"
-            
+            id="contact-me-form"
+            name="contactMe"
```

Comment:
- [ ]  I noticed that the input fields in the contact form do not allow me to enter any information. Please try to fix this issue in order to get approved ✔️

Reply:
@codepantha Okay